domains/trevor.json

```
@@ -0,0 +1,15 @@
+{
+  "owner": {
+    "username": "TrevorDBrown",
+    "email": "trevordylanbrown@gmail.com"
+  },
+  "records": {
+    "CNAME": "trevord.me"
```

```suggestion
    "URL": "trevord.me"
```


URL records redirect (does not change your url/link)